1-JavaScript/Week2/homework/ex6-totalCost.js

```diff
@@ -21,22 +21,35 @@ instead!
 3. Complete the unit test functions and verify that all is working as expected.
 -----------------------------------------------------------------------------*/
 const cartForParty = {
-  // TODO complete this object
+beer:1.8,
+chips:2.2,
+cake:1.2,
+iceCream:3.4,
+juice:0.99
```

Comment:
Ice cream, nice! 🍦 What is your favourite? 😄 

I would recommend to also add some spaces here, for readability.

```suggestion
beer: 1.8,
chips: 2.2,
cake: 1.2,
iceCream: 3.4,
juice: 0.99
```